packages/vertexai/src/types/responses.ts

```diff
@@ -74,6 +74,11 @@ export interface GenerateContentResponse {
   usageMetadata?: UsageMetadata;
 }
 
+/**
+ * Usage metadata about response.
```

Comment:
nit `a response`

Reply:
Could even link to `GenerateContentResponse`, will do that.